src/python/pants/core/subsystems/python_bootstrap.py

```diff
@@ -89,7 +89,7 @@ class PythonBootstrapSubsystem(Subsystem):
 
     class EnvironmentAware(Subsystem.EnvironmentAware):
         search_path = StrListOption(
-            default=["<PYENV>", "<PATH>"],
+            default=["<PYENV>", f"{AsdfPathString.STANDARD}", "<PATH>"],
```

Comment:
```suggestion
            default=["<PYENV>", AsdfPathString.STANDARD, "<PATH>"],
```

Reply:
I agree with Benjy on this one in the sense that `f"{...}"` looks like a code smell.
Perhaps this could work?
```suggestion
            default=["<PYENV>", AsdfPathString.STANDARD.value, "<PATH>"],
```

Reply:
That works +1